README.md

```diff
@@ -4,6 +4,20 @@ A frontend for creating and managing email subscriptions.
 
 ## Overview
 
+### Running the application
+
+```sh
+$ ./startup.sh
+```
+
+If you encounter 404 pages, you can also use [govuk-docker](https://github.com/alphagov/govuk-docker) adding the `--live` flag to use live data 
+
+```sh
+$ govuk-docker up email-alert-frontend-app-live
+```
+
+If you are using the GDS development virtual machine then the application will be available on the host at http://email-alert-frontend.dev.gov.uk/
```

Comment:
Thanks for suggesting this.

This guidance seems very specific towards running this app in isolation of the rest of the GOV.UK stack - which might be the common case for a frontend developer but not necessarily what a backend developer wants.

This guidance was recently removed in https://github.com/alphagov/email-alert-frontend/commit/d6de99f6b1104f8e02a088f394ee41cf287c66f1#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 - with this approach applied across the GOV.UK stack. 

So rather than a small change this is probably part of a wider conversation about what should be in GOV.UK applications READMEs.

Reply:
> This guidance seems very specific towards running this app in isolation of the rest of the GOV.UK stack 

I'm wondering if there's a better way to phrase this?  In this particular context the app _was_ run in isolation.  

I'm thinking there might be a concern about noise within this doc? Wondering if there's a halfway to reduce the length but still include the info that would have been helpful.

> which might be the common case for a frontend developer but not necessarily what a backend developer wants.

Is this app generally speaking, only used backend developers? Is more documentation problematic in some way? 
>So rather than a small change this is probably part of a wider conversation about what should be in GOV.UK applications READMEs.

Maybe yes, it might be nice to have more of a generic README Template that's consistent across all apps that includes startup in isolation.

Reply:
>  Is more documentation problematic in some way?

More isn't problematic, but misleading is. One of the challenges we've had (and why the existing run guides were deleted) was that they suggested people do things in a way that wasn't consistent with the govuk-docker recommended approach, which means someone would follow the documentation and it wouldn't work.

Noise isn't so much a concern it's more about repeating things that could age badly - for example listing the govuk-docker commands in this README couples this documentation to any changes in govuk-docker.

I'm guessing what we'd want to say to tell someone how to run this application is:

```
### Running this application

This application is designed to be run within the [GOV.UK Docker](https://github.com/alphagov/govuk-docker) development environment.
```

The `startup.sh` scripts are caught as a bit of an anomaly since the retirement of the development VM. Removing them would definitely cause people problems so that isn't currently considered but it doesn't seem great to be suggesting two different approaches to running an app and making a developer choose.

It's probably worth breaking this change into a separate PR so it doesn't block the heading change.

Reply:
>This application is designed to be run within the [GOV.UK Docker](https://github.com/alphagov/govuk-docker) development environment.

This works, can we add anything within this copy to mentioned the `--live` flag?  As this was the tiny bit of crucial info that (I don't think..) is required for all apps?

Perhaps this can be updated on [this docker page](https://github.com/alphagov/govuk-docker/blob/master/docs/compatibility.md)? I'm trying to think of a way - coming into this without touching anything else for the first time - this can be communicated